src/java/src/test/java/com/dnanexus/DXSearchTest.java

```diff
@@ -597,6 +597,117 @@ public void testFindDataObjectsWithPaging() {
     }
 
     /**
+     * Tests paging through results (new pagination API).
+     */
+    @Test
+    public void testFindDataObjectsPaginated() {
+        List<DXRecord> records = Lists.newArrayList();
+        Set<String> recordIds = Sets.newHashSet();
+        for (int i = 0; i < 8; ++i) {
+            DXRecord record =
+                    DXRecord.newRecord().setProject(testProject)
+                            .setName("foo" + Integer.toString(i)).build();
+            records.add(record);
+            recordIds.add(record.getId());
+        }
+        List<DXRecord> outputRecords =
+                DXSearch.findDataObjects().inProject(testProject).nameMatchesGlob("foo*")
+                        .withClassRecord().execute().asList();
+        Assert.assertEquals(8, outputRecords.size());
+
+        List<DXRecord> outputRecordsWithPaging =
```

Comment:
Seems like outputRecordIds below isn't used, and in any case this entire block seems to duplicate code in another test. Can this block be removed entirely?